e2e/pages/adf/filePreviewPage.ts

```diff
@@ -89,16 +89,10 @@ export class FilePreviewPage {
 
         this.checkCanvasWidth().then((width) => {
```

Comment:
checkCanvasWidth() method should be renamed to getCanvasWidth(), since it's not checking anything, just returning a value. Same for checkCanvasHeight().